### PR TITLE
build: move organizers and community support to hygraph

### PR DESCRIPTION
**Pull request by antoncoding**
* fetch data from hygraph directly
* community support entities are also stored in the "partner" scheme. Only with a flag `isCommunitySupport == true`

### CODE REVIEW

**Review by DylanYang0523 — APPROVED** [review]
LGTM!

File: styles/colors.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ const Colors = {
   pennBlue: "#0A1E4E",
   aero: "#59B2D6",
   yInMnBlue: "#f0f5ff",
+  lightBlue2: "#cbdff7",
```
Comment (Ankarrr) [inline styles/colors.ts:12]: 為什麼不用跟其他 section 同樣顏色？

Comment (Ankarrr) [reply styles/colors.ts:12]: 看留言應該是因為 ethseoul logo，但我覺得有一個 section 顏色不同怪怪的 XD

**Review by Ankarrr — APPROVED** [review]

File: styles/colors.ts (hunk 1 above)
Comment (antoncoding) [reply styles/colors.ts:12]: 那我還是先不理他們好了，感覺總體觀感有扣份

**Pull request merged**